### PR TITLE
Honor EMSDK_PYTHON in .bat file launcher scripts

### PR DESCRIPTION
**Pull request by sbc100**
This change is mostly auto-generated by `tools/create_entry_points.py`

### CODE REVIEW

**Comment by kripken** [comment]
Is one of the commits auto-generated and one not?

File: em++.bat [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @echo off
-REM Entry point for running python scripts on windows systems.
-REM To modify this file, edit `tools/run_python.bat` and then run
-REM `tools/create_entry_points.py`
+:: Entry point for running python scripts on windows systems.
+:: To modify this file, edit `tools/run_python.bat` and then run
+:: `tools/create_entry_points.py`
```
Comment (kripken) [inline em++.bat:4]: isn't it `REM` though? that's what I remember... but it's been a long time.

Comment (sbc100) [reply em++.bat:4]: I noticed this style in emsdk.bat file so I thought I would be consistent with that.

Apparently they are both used for comments with this method being slightly faster:  https://www.robvanderwoude.com/comments.php

**Comment by sbc100** [comment]
I modified the two template files and then ran the script.. so the changes at the top level are all auto-generated.

**Pull request merged**